src/Microsoft.AspNet.Diagnostics.Entity/DatabaseErrorPageExtensions.cs

```diff
@@ -27,38 +26,9 @@ public static IApplicationBuilder UseDatabaseErrorPage(this IApplicationBuilder
                 throw new ArgumentNullException(nameof(app));
             }
 
-            return app.UseDatabaseErrorPage(options => options.EnableAll());
-        }
-
-        /// <summary>
-        /// Captures synchronous and asynchronous database related exceptions from the pipeline that may be resolved using Entity Framework
-        /// migrations. When these exceptions occur an HTML response with details of possible actions to resolve the issue is generated.
-        /// </summary>
-        /// <param name="app">The <see cref="IApplicationBuilder"/> to register the middleware with.</param>
-        /// <param name="configureOptions">An action to set the options for the middleware. All options are disabled by default.</param>
-        /// <returns>The same <see cref="IApplicationBuilder"/> instance so that multiple calls can be chained.</returns>
-        public static IApplicationBuilder UseDatabaseErrorPage(this IApplicationBuilder app, Action<DatabaseErrorPageOptions> configureOptions)
-        {
-            if (app == null)
-            {
-                throw new ArgumentNullException(nameof(app));
-            }
-            if (configureOptions == null)
-            {
-                throw new ArgumentNullException(nameof(configureOptions));
-            }
-
             var options = new DatabaseErrorPageOptions();
-            configureOptions(options);
-
-            app = app.UseMiddleware<DatabaseErrorPageMiddleware>(options);
-
-            if (options.EnableMigrationCommands)
-            {
-                app.UseMigrationsEndPoint(o => o.Path = options.MigrationsEndPointPath);
-            }
-
-            return app;
+            options.EnableAll();
```

Comment:
Whatever we do here should just be consistent with the developer exception page. I assume the developer exception page defaults to showing everything.